app/views/admin/index.slim

```diff
@@ -0,0 +1,21 @@
+.row.character_index_page
+	.small-12.small-centered.columns
+
+		h1 Admin
```

Comment:
The `h1` tag should only be used for the main header of the site. This should therefore be an `h2`